src/tests/e2etest/E2ETest.cs

```diff
@@ -201,28 +204,28 @@ public bool UpdateTwinBlockUntilUpdate(Twin patch)
 
             if (UpdateTwinBlockUntilUpdate(twinPatch))
             {
-                var deserializedTwin = JsonSerializer.Deserialize<CommandRunnerTests.CommandStatus>(GetTwin().Properties.Reported["CommandRunner"]["CommandStatus"].ToString());
+                var deserializedObject = JsonSerializer.Deserialize<CommandRunnerTests.CommandStatus>(GetTwin().Properties.Reported["CommandRunner"]["CommandStatus"].ToString());
                 // Wait until commandId is equivalent
                 DateTime startTime = DateTime.Now;
-                while(deserializedTwin.CommandId != random && (DateTime.Now - startTime).TotalSeconds < 30)
+                while(deserializedObject.CommandId != random && (DateTime.Now - startTime).TotalSeconds < 30)
                 {
                     Console.WriteLine("[PerformCommandViaCommandRunner] waiting for commandId to be equivalent...");
                     Task.Delay(twinRefreshIntervalMs).Wait();
-                    deserializedTwin = JsonSerializer.Deserialize<CommandRunnerTests.CommandStatus>(GetNewTwin().Properties.Reported["CommandRunner"]["CommandStatus"].ToString());
+                    deserializedObject = JsonSerializer.Deserialize<CommandRunnerTests.CommandStatus>(GetNewTwin().Properties.Reported["CommandRunner"]["CommandStatus"].ToString());

```

Comment:
>GetNewTwin [](http://example.com/codeflow?start=102&length=10)

These method names should be changed from Get*Twin to Get*Reported or similar. They get an object, not a twin.

Reply:
Since the base APIs use the name "twin" let's keep these as they are.